packages/playground/windows/playground/MainPage.cpp

```diff
@@ -73,6 +73,24 @@ void MainPage::OnLoadClick(
     host.InstanceSettings().DebugHost(m_bundlerHostname);
   }
 
+  host.InstanceSettings().InstanceCreated(
```

Comment:
Slightly tangential, but I remember it tripping me up when working on integration tests that lifecycle events were fired off UI thread.

Wonder if we could find a way to make thread affinity clearer. I kind of like what RNS does where it explicitly asks for a dispatcher.

Reply:
Adding a dispatcher would make the threading more complex.  We will be relying on this event being synchronous so that we can inject things into the JS context before the main bundle is run in Office.  It's an important hook in the instance boot cycle.  But I agree we should ensure that the documentation includes threading expectations of this event.